### ISSUE TITLE
拜托回我吧

### ISSUE THREAD

**Issue by Bluelyy** [GXYM/DRRG#22]
训练了ctw1500 但是那几个数据集比如icdar2015 等  都没有训练以及测试文件  没法训练和测试  是文件上传少了一部分吗 进行不下去了  在线等...

**Comment by GXYM** [comment]
We don't provide the data of each training set and test set, which need to be prepared by you.

**Comment by GXYM** [comment]
> 训练了ctw1500 但是那几个数据集比如icdar2015 等 都没有训练以及测试文件 没法训练和测试 是文件上传少了一部分吗 进行不下去了 在线等...

We don't provide the data of each training set and test set, which need to be prepared by you.
You just need to prepare the data, and the training and testing code of the model for these data sets is complete.
Please refer to:
https://github.com/GXYM/DRRG/tree/master/dataset
https://github.com/GXYM/DRRG/blob/master/util/option.py
https://github.com/GXYM/DRRG/blob/master/train_TextGraph.py
https://github.com/GXYM/DRRG/blob/master/eval_TextGraph.py
